### PR TITLE
parse raw kafka data for sensors like li7200

### PR DESCRIPTION
**Pull request by liangyingz**

### CODE REVIEW

File: pipe/li7200/li7200_data_source_trino.yaml [hunk 1]
```diff
@@ -45,7 +45,7 @@ transform:
       export GEN_OUTPUT_DIR=$interimDir/$SOURCE_TYPE/$year/$month/$day
       export REQUESTS_CA_BUNDLE=/etc/pki/tls/cert.pem
       mkdir -p $GEN_OUTPUT_DIR
-      /usr/src/app/genscript/genparquet.py --storesitename --codec gzip
+      /usr/src/app/genscript/genparquet.py --storesitename --codec gzip --truncperiod second
```
Comment (covesturtevant) [inline pipe/li7200/li7200_data_source_trino.yaml:48]: Are you sure you want to truncate to the second? Isn't li-7200 code at 20 Hz?

File: modules/raw_data_parser/build_tag_push_update.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Run from root repository (NEON-IS-data-processing)
+#!/usr/bin/env bash
+image_name=raw-data-parser
```
Comment (covesturtevant) [inline modules/raw_data_parser/build_tag_push_update.sh:3]: As we move to the GCP image registry, we'll ensure all image names start with "neon-is". Please migrate to this naming convention at your convenience.

**Review by covesturtevant — DISMISSED** [review]
This looks great. Feel free to merge, but please see comments for minor concerns.

File: pipe/li7200/li7200_data_source_trino.yaml (hunk 1 above)
Comment (liangyingz) [reply pipe/li7200/li7200_data_source_trino.yaml:48]: Good catch, will revert

File: modules/raw_data_parser/build_tag_push_update.sh (hunk 2 above)
Comment (liangyingz) [reply modules/raw_data_parser/build_tag_push_update.sh:3]: A quay.io repo is already created. I will keep this in mind and make the change before we move to the GCP image registry.

File: pipe/li7200/li7200_data_source_trino.yaml (hunk 1 above)
Comment (liangyingz) [reply pipe/li7200/li7200_data_source_trino.yaml]: seems need your approval again with a new check-in. Thanks!

**Comment by liangyingz** [comment]
@covesturtevant seems need your approval again with a new check-in. Thanks!

**Review by covesturtevant — APPROVED** [review]

**Pull request merged**